packages/editor-ui/src/mixins/pushConnection.ts

```diff
@@ -285,10 +286,17 @@ export const pushConnection = defineComponent({
 
 				let runDataExecutedErrorMessage = this.getExecutionError(runDataExecuted.data);
 
-				if (pushData.data.status === 'crashed') {
+				if (runDataExecuted.status === 'crashed') {
 					runDataExecutedErrorMessage = this.$locale.baseText(
 						'pushConnection.executionFailed.message',
 					);
+				} else if (runDataExecuted.status === 'canceled') {
+					runDataExecutedErrorMessage = this.$locale.baseText(
+						'executionsList.showMessage.stopExecution.message',
```

Comment:
We're using `canceled` internally but `stopped` externally - we might want to unify for consistency.

Reply:
Agreed that we should unify these. Created a ticket [ENG-93](https://linear.app/n8n/issue/ENG-93)